packages/lodestar/src/metrics/metrics/beacon.ts

```diff
@@ -126,5 +126,21 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       help: "Time taken to find head in seconds",
       buckets: [0.1, 1, 10],
     }),
+    forkChoiceHeadRequests: register.gauge({
+      name: "beacon_fork_choice_requests_total",
+      help: "Count of occasions where fork choice has tried to find a head",
+    }),
+    forkChoiceErrors: register.gauge({
+      name: "beacon_fork_choice_errors_total",
+      help: "Count of occasions where fork choice has returned an error when trying to find a head",
+    }),
+    forkChoiceNewHeads: register.gauge({
```

Comment:
Can you make the name match the metric name for consistency?
- `forkChoiceRequests`
- `forkChoiceChangedHead`

Reply:
ok, no problem